instaloader/structures.py

```diff
@@ -11,11 +11,12 @@
 from .instaloadercontext import InstaloaderContext
 from .nodeiterator import FrozenNodeIterator, NodeIterator
 
-PostSidecarNode = namedtuple('PostSidecarNode', ['is_video', 'display_url', 'video_url'])
+PostSidecarNode = namedtuple('PostSidecarNode', ['is_video', 'display_url', 'video_url', 'is_last'])
```

Comment:
The field `is_last` does not make any sense here. It would be better to let the caller of `Post.get_sidecar_nodes()` determine which node is the last one.

Reply:
I've tried a different approach: do you thing it's better?